src/pipeline_edsl/to_workflow.ml

```diff
@@ -539,7 +539,7 @@ module Make (Config : Compiler_configuration)
 
   let vcf_annotate_polyphen reference_build vcf =
     let v = get_vcf vcf in
-    let out_filename = Filename.chop_extension v#product#path ^ "_polyphen.vcf" in
+    let out_filename = Filename.basename v#product#path ^ "_polyphen.vcf" in
     let output_vcf = Config.work_dir // out_filename in 
```

Comment:
I think it's better to remove the `Config.work_dir // ` part.
In the EDSL we already ensure that everything is in the work-directory since we control how things are "input" at their source (right now only `fastq`, `fastq_gz` and `bam`; they have the option to copy, `ln -s`, or let the user mess up).

`Filename.basename` removes the directory part so this is going to make files like `somethingsomethingsomething.vcf_polyphen.vcf`.